packages/vite-plugin/src/hot-update.ts

```diff
@@ -63,8 +63,8 @@ function patchModule(
     const oCompFns = oVineCompFns[i]
     const nCompFnsTemplate = normalizeLineEndings(nCompFns.templateSource)
     const oCompFnsTemplate = normalizeLineEndings(oCompFns.templateSource)
-    const nCompFnsStyle = normalizeLineEndings(nStyleDefine[nCompFns.scopeId].source)
-    const oCompFnsStyle = normalizeLineEndings(oStyleDefine[oCompFns.scopeId].source)
+    const nCompFnsStyle = normalizeLineEndings((nStyleDefine[nCompFns.scopeId] || { source: '' }).source)
+    const oCompFnsStyle = normalizeLineEndings((oStyleDefine[oCompFns.scopeId] || { source: '' }).source)
```

Comment:
```suggestion
    const nCompFnsStyle = normalizeLineEndings(nStyleDefine[nCompFns.scopeId]?.source ?? '')
    const oCompFnsStyle = normalizeLineEndings(oStyleDefine[oCompFns.scopeId]?.source ?? '')
```

Maybe we can use more neat form for this fallback ?